cpp/src/column/column_view.cpp

```diff
@@ -45,7 +45,10 @@ void prefetch_col_data(ColumnView& col, void const* data_ptr, std::string_view k
         key, data_ptr, col.size() * size_of(col.type()), cudf::get_default_stream());
     } else if (col.type().id() == type_id::STRING) {
       strings_column_view scv{col};
-
+      if (data_ptr == nullptr) {
+        // Do not call chars_size if the data_ptr is nullptr.
+        return;
+      }
```

Comment:
This was a bug. We can't call `chars_size` on string column views that are not fully initialized like this one: https://github.com/rapidsai/cudf/blob/8def2ec1acac6a538002db011d977bb22cfbda82/cpp/src/column/column_factories.cu#L60-L62

Errors from calling `chars_size()` looked like this:
```
terminate called after throwing an instance of 'cudf::logic_error'
  what():  CUDF failure at: /home/coder/cudf/cpp/src/strings/strings_column_view.cpp:34: strings column has no children
Aborted (core dumped)
```